include/seqan3/range/view/all.hpp

```diff
@@ -114,48 +114,97 @@
+ * `auto vec_view` above)
+ *   2. the view adaptor (this is the functor that returns the actual view based on it's parameters, including the
+ * underlying range; in the above example `ranges::view::reverse` and `view::complement` are view adaptors)
+ *
+ * The view adaptor also facilitates the piping behaviour. It is the only entity that is publicly documented and
+ * the actual view type (the range type returned by the adaptor) is considered implementation defined.
+ * The *properties* of the returned range are however specified and documented as part of the adaptor, see below.
+ *
+ * <sub>
+ * An exception to this rule are views that don't work on an underlying range and can only be
+ * placed at the beginning of a pipe of operations; they do not need an adaptor, because their constructor is
+ * sufficient. This is not relevant for the documention, though, we always document `view::foo`, independent of
+ * whether `view::foo` is the adaptor that returns the "foo type" or whether `view::foo` is the "foo type".
+ * </sub>
+ *
+ * ### View properties
+ *
+ * There are three view properties that are documented for a view, **only if that view fulfills them:**
+ *
+ * **Source-only views:** Most views operate on an underlying range and return a (modified) range, i.e. they can be placed
  * at the beginning, middle or end of a "pipe" of view operations. However, some views are limited to being at
- * the front ("source"), e.g. `ranges::view::single`, `ranges::view::concat` and `ranges::view::ints`.
+ * the front ("source"), e.g. `ranges::view::single`, `ranges::view::concat` and `ranges::view::ints`. These views
+ * are marked as "source-only" and have no `urng_t` column in the second table.
  *
  * **Sink-only views:** The opposite of a *source-only view*. It can only be placed at the end of a pipe, i.e.
- * it operates on views, but does not actually return a view.
+ * it operates on views, but does not actually return a range (has no `rrng_t` column in the second table).
  *
  * **Deep views:** Some views are declared as "deeps views". This means, that in case they are given a range-of-range
  * as input (as opposed to just a range), they will apply their transformation on the innermost range (instead of
  * the outermost range which would be default). This is handy especially for alphabet-based transformations that you
  * wish to apply to a collection of sequences.
  *
- * **Input range requirements:** All views that are not *source-only views* make certain assumptions about their input.
+ * **For all views the following are documented:**
+ *
+ * | range concepts and reference_t      | `urng_t` (underlying range type) | `rrng_t` (returned range type)                     |
+ * |-------------------------------------|----------------------------------|----------------------------------------------------|
+ * | seqan3::input_range_concept         | [required] <i>(usually)</i>      | [preserved\|lost\|guaranteed] (usually preserved)  |
```

Comment:
why suddenly using html markup instead of underscores for markdown.

Reply:
It didn't work for some reason in the table :cry: